### PR TITLE
Add Cav Scout Characters

### PR DESCRIPTION
**Pull request by Robot-Panda15**
**When merged this pull request will:**
Part of #35 
(Brostrom, might want to look at the displayName section of these)

### CODE REVIEW

File: addons/bravo_characters/CfgVehicles.hpp [hunk 1]
```diff
@@ -174,6 +228,16 @@ class CfgVehicles {
 
         icon = "iconManLeader";
     };
+    class Cav_B_Bravo_Scout_VC_F : Cav_B_Bravo_Scout_base_F {
```
[the remark this answers is not in the document]
Comment (Robot-Panda15) [reply addons/bravo_characters/CfgVehicles.hpp]: VC is Vehicle Commander

File: addons/bravo_characters/CfgVehicles.hpp [hunk 2]
```diff
@@ -34,6 +34,40 @@ class CfgVehicles {
             "rhsusf_ANPVS_14"
         };
     };
+        class Cav_B_Bravo_Scout_base_F : Cav_B_Bravo_base_F {
```
Comment (ColdEvul) [inline addons/bravo_characters/CfgVehicles.hpp]: Indent

**Comment by ColdEvul** [comment]
The line indent else everything looks great! :)

**Comment by ColdEvul** [comment]
@Robot_Panda15#9906
have you tested this? 
If so I'll merge this. :)

**Comment by Robot-Panda15** [comment]
Haven't had a chance to as of yet

**Comment by ColdEvul** [comment]
Fixed in #56 Tho its more or less a rollback

**Comment by ColdEvul** [comment]
Handle by #69 
Closed

**Pull request closed without merging**